doc/source/user_guide/offline/file_session.rst

```diff
@@ -33,7 +39,7 @@ Single-phase
   >>> file_session.read_data(data_file_name)
```

Comment:
@prmukherj
- It looks like the example forgot to instantiate this object.
- It would be a nice improvement to be able to specify the input files while instantiating `FileSession` objects as opposed to having to call the `read_` methods. Given the example code, I assume it's not yet possible.

Reply:
Done, Thank you.